src/filesystem/interface.js

```diff
@@ -1,67 +1,70 @@
-var { promisify } = require('es6-promisify');
-
-var Path = require('../path.js');
-var nop = require('../shared.js').nop;
-
-var Constants = require('../constants.js');
-var FILE_SYSTEM_NAME = Constants.FILE_SYSTEM_NAME;
-var FS_FORMAT = Constants.FS_FORMAT;
-var FS_READY = Constants.FS_READY;
-var FS_PENDING = Constants.FS_PENDING;
-var FS_ERROR = Constants.FS_ERROR;
-var FS_NODUPEIDCHECK = Constants.FS_NODUPEIDCHECK;
-
-var providers = require('../providers/index.js');
-
-var Shell = require('../shell/shell.js');
-var Intercom = require('../../lib/intercom.js');
-var FSWatcher = require('../fs-watcher.js');
-var Errors = require('../errors.js');
-var defaultGuidFn = require('../shared.js').guid;
-
-var STDIN = Constants.STDIN;
-var STDOUT = Constants.STDOUT;
-var STDERR = Constants.STDERR;
+'use strict';
+
+const { promisify } = require('es6-promisify');
+
+const Path = require('../path.js');
+const nop = require('../shared.js').nop;
+
+const providers = require('../providers/index.js');
+
+const Constants = require('../constants.js');
+const Shell = require('../shell/shell.js');
+const Intercom = require('../../lib/intercom.js');
+const FSWatcher = require('../fs-watcher.js');
+const Errors = require('../errors.js');
+const defaultGuidFn = require('../shared.js').guid;
+
+const {
+  FILE_SYSTEM_NAME,
+  FS_FORMAT,
+  FS_READY,
+  FS_PENDING,
+  FS_ERROR,
+  FS_NODUPEIDCHECK,
+  STDIN,
+  STDOUT,
+  STDERR
+} = Constants;
 
 // The core fs operations live on impl
-var impl = require('./implementation.js');
+const impl = require('./implementation.js');
 
 // node.js supports a calling pattern that leaves off a callback.
 function maybeCallback(callback) {
-  if(typeof callback === 'function') {
+  if (typeof callback === 'function') {
```

Comment:
In general, be careful of this kind of thing (i.e., changing formatting in lines unrelated to your actual changes).  We call this a "spurious whitespace change" and open source projects tend to want to avoid it, since it masks the history of this line of code, making it look like it was changed here by you, when actually it was just a formatting fix.

I'll allow it this time, but be aware that it's not generally a good idea.  If your editor did it automatically, I'd disable that.